WISE-Community/plate-tectonics-boundary-explorer
# UI/UX improvements

## Changes
- Improved accessibility:
  - Added keyboard navigation support
  - Added 'alt' attributes to all images
  - Added 'aria-label' when necessary to buttons
- Reordered prediction tasks for more logical UX
- Updated some text
- Updated layout and styles to better support screen resizing

Closes #3. 

## Test
- Make sure model works as before
- Make sure there are no visual bugs or problems
